velox/vector/tests/CopyPreserveEncodingsTest.cpp

```diff
@@ -327,10 +327,11 @@ TEST_F(CopyPreserveEncodingsTest, flatNoNulls) {
 TEST_F(CopyPreserveEncodingsTest, lazyNoNulls) {
   auto lazyVector = vectorMaker_.lazyFlatVector<int32_t>(
       10, [](vector_size_t row) { return row; });
+  auto copy = lazyVector->loadedVector()->copyPreserveEncodings();
```

Comment:
Maybe add a check that if someone trying to copy unloaded lazy, they still get an exception